releases.yml

```diff
@@ -1,9 +1,15 @@
+- version: 4.27.0
+  features:
+    - component: Grid
+      url: /docs/patterns/grid#common-patterns-new
+      status: New
+      notes: We've added common grid pattern shorthands for the new grid.
 - version: 4.26.0
   features:
     - component: Icons
       url: /docs/patterns/icons#recently-added-icons-new
       status: New
-      notes: We've added the new <code>p-icon--history</code><i class="p-icon--history"/> icon.
```

Comment:
drive-by fix for this mis-styled release note on the [what's new page](https://vanillaframework.io/docs/whats-new). p-icon-- classes aren't in scope on this page

<img width="410" height="64" alt="image" src="https://github.com/user-attachments/assets/17da8b16-b0ee-412f-8aa0-0ff10e4fa0a5" />